pkg/services/object/put/distributed.go

```diff
@@ -134,15 +135,28 @@ func (t *distributedTarget) Close() (oid.ID, error) {
 
 	t.obj.SetPayload(t.payload)
 
+	if len(t.obj.Children()) > 0 || t.obj.Type() == objectSDK.TypeLink {
+		// enabling extra broadcast for linking objects
+		t.traversalParams.extraBroadcastEnabled = true
+	}
+
+	id, _ := t.obj.ID()
+	t.traversalParams.opts = append(t.traversalParams.opts, placement.ForObject(id))
 	var err error
 
-	if t.objMeta, err = t.fmt.ValidateContent(t.obj); err != nil {
-		return oid.ID{}, fmt.Errorf("(%T) could not validate payload content: %w", t, err)
+	t.traverser, err = placement.NewTraverser(t.traversalParams.opts...)
+	if err != nil {
+		return oid.ID{}, fmt.Errorf("(%T) could not create object placement traverser: %w", t, err)
 	}
 
-	if len(t.obj.Children()) > 0 || t.obj.Type() == objectSDK.TypeLink {
-		// enabling extra broadcast for linking objects
-		t.traversal.extraBroadcastEnabled = true
+	// v2 split link object validation is an expensive routine and useless if
+	// the node does not belong to the container, since another node is responsible
+	// for the link validation and may decline it, does not matter what this node
+	// thinks about it
+	if t.obj.Type() != objectSDK.TypeLink || t.traverser.KeyInContainer(t.isLocalKey) {
+		if t.objMeta, err = t.fmt.ValidateContent(t.obj); err != nil {
```

Comment:
I think some syntactic check can still be made here, although it doesn't seem to be critical.

Reply:
you mean to try to parse the link object but not validate? or what "syntactic check"?

Reply:
Yes, parse, but don't go fetching/checking things.